lib/active_resource/log_subscriber.rb

```diff
@@ -3,6 +3,7 @@ class LogSubscriber < ActiveSupport::LogSubscriber
     def request(event)
       result = event.payload[:result]
       info "#{event.payload[:method].to_s.upcase} #{event.payload[:request_uri]}"
+      info "Params sent: #{event.payload[:params].inspect}" if event.payload[:params].present?
```

Comment:
Would not this show sensitive information if the parameters contain them? Maybe we should pass them though some kind of parameters filtering?